course/layouts/index.coursedata.json

```diff
@@ -29,6 +30,7 @@
   "extra_course_numbers":  {{- $courseData.extra_course_numbers | jsonify -}},
   "term":  {{- $courseData.term | jsonify -}},
   "year":  {{- $courseData.year | jsonify -}},
-  "level":  {{- $courseData.level | jsonify -}}
+  "level":  {{- $courseData.level | jsonify -}},
+  "image_src": {{- $courseImageUrl | jsonify -}}
```

Comment:
When I ran this locally I got
  "image_src":"/15-401-finance-theory-i-fall-2008/5ec2e66e89b03d5c5c7806672a26167e_15-401f08.jpg"
I think you want to just return  $courseImageMetadata.Params.file in the json files instead of the relative url

Reply:
that's odd - the logic I used is the same as what's used for rendering the course image on the home page. do you see the same url showing up there in the rendered template?